lstchain/image/muon/muon_analysis.py

```diff
@@ -125,13 +127,29 @@ def fit_muon(x, y, image, geom, tailcuts):
     image_clean: `np.ndarray`
         Image after cleaning
     """
+    
+    if tailcuts == None:
+        #we want to quantize the noise of the image. To do so, we will use the negative Q cumulative distribution.
+        negative_Q=np.sort(image[image <= 0])
```

Comment:
Add spaces around  "="

Also: np.sort is not needed here, because you histogram the values, and only use the bin contents after that.

Reply:
done in #1112